2026-03-16-release-1.26.md

```
@@ -0,0 +1,353 @@
+
+---
+
+## Community meeting
+
+If you would like to discuss any of the new features, issues or other topics, please join us at the monthly community meeting that takes place <b>every 4th Thursday of the month</b>. Here are the <b>meeting [details](https://github.com/podman-desktop/community/issues?q=is%3Aissue%20state%3Aopen%20Agenda%20for%20Podman%20Desktop)</b>.
+
+---
+
+## Final notes
+
+The complete list of issues fixed in this release is available [here](https://github.com/containers/podman-desktop/issues?q=is%3Aclosed+milestone%3A1.26.0) and [here](https://github.com/containers/podman-desktop/issues?q=is%3Aclosed+milestone%3A1.26.0).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)